_includes/header.html

```diff
@@ -19,13 +19,7 @@
                 <a class="nav-link {% if page.url == '/subscribe/' %}active{% endif %}" href="/subscribe/">subscribe</a>
             </li>
             <li class="nav-item mr-2">
-                <a class="nav-link {% if page.url == '/search/' %}active{% endif %}" href="/search/">search</a>
-            </li>
-            <li class="nav-item mr-2">
-                <a class="nav-link" href="{{ site.links.github }}">github</a>
-            </li>
-            <li class="nav-item mr-2">
-                <a class="nav-link" href="https://twitter.com/{{ site.author.twitter }}">twitter</a>
+                <a class="nav-link" href="https://twitter.com/{{ site.author.twitter }}"><i class="fab fa-twitter"></i></a>
```

Comment:
I'd be down for a search icon in the nav bar -- but I don't feel strongly about that.

Reply:
Done.